src/qt/bitcoin.cpp

```diff
@@ -315,11 +315,9 @@ void BitcoinApplication::parameterSetup()
     InitParameterInteraction(gArgs);
 }
 
-void BitcoinApplication::InitializePruneSetting(bool prune)
+void BitcoinApplication::InitPruneSetting(int64_t prune_opt)
```

Comment:
273e57bd4cd91e8399f3567c79241bbc416ea690

Why the rename? To ensure calls are refactored?

Reply:
Yes, otherwise a bool might get silently treated as a 1